jsk_recognition_msgs/msg/Label.msg

```diff
@@ -0,0 +1,3 @@
+int32 index
```

Comment:
I think `label` or `id` is more appropriate name, because `index` is usually non-negative and it must be with a list of values.